File: app/src/main/java/me/rhunk/snapenhance/XposedLoader.java
package me.rhunk.snapenhance;

import de.robv.android.xposed.IXposedHookLoadPackage;
import de.robv.android.xposed.XposedBridge;
import de.robv.android.xposed.XposedHelpers;
import de.robv.android.xposed.callbacks.XC_LoadPackage;

public class XposedLoader implements IXposedHookLoadPackage {
    @Override
    public void handleLoadPackage(XC_LoadPackage.LoadPackageParam packageParam) throws Throwable {
        if (!packageParam.packageName.equals(Constants.SNAPCHAT_PACKAGE_NAME)) return;
        new SnapEnhance();
    }
}

File: core/src/main/kotlin/me/rhunk/snapenhance/XposedLoader.kt
package me.rhunk.snapenhance

import de.robv.android.xposed.IXposedHookLoadPackage
import de.robv.android.xposed.callbacks.XC_LoadPackage

class XposedLoader : IXposedHookLoadPackage {
    override fun handleLoadPackage(p0: XC_LoadPackage.LoadPackageParam) {
        if (p0.packageName != Constants.SNAPCHAT_PACKAGE_NAME) return
        SnapEnhance()
    }
}